frontend/src/components/RecipeCardContainer/RecipeCardContainer.jsx

```
@@ -101,50 +101,67 @@ const RecipeCardContainer = ({
-            );
-          })
-        )}
-      </Grid>
+              ) : (
+                <Flex
+                  w="100vw"
+                  h="lg"
+                  alignItems="center"
+                  justifyContent="center"
+                  direction="column"
+                >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)